k8s/pkg/tests/registry_cache_test.go

```diff
@@ -30,15 +30,21 @@ func fakeNsmRest(serverData *sync.Map) *FakeRest {
 	result.MockGet("/networkserviceendpoints", func(r *http.Request, resource string) (response *http.Response, e error) {
 		return Ok([]v1.NetworkServiceEndpoint{}), nil
 	})
+	result.MockGet("/namespaces/default/networkserviceendpoints", func(r *http.Request, resource string) (response *http.Response, e error) {
+		return Ok(v1.NetworkServiceEndpointList{}), nil
+	})
 	result.MockGet("/namespaces/default/networkservicemanagers", func(r *http.Request, resource string) (response *http.Response, e error) {
 		if val, ok := serverData.Load(resource); ok {
 			return Ok(val), nil
 		}
-		return NotFound(v1.NetworkServiceManager{}), nil
```

Comment:
Why we need change this?

Reply:
As far as I know NotFound means there is no such type of resource as NSM. It differs from empty list of NSM